javascript-data-model/order-history.js

```diff
@@ -0,0 +1,67 @@
+// const { PureComponent } = require("react");
+
+var orderHistory = [
+  {
+    order_placed: { month: 'August', day: 4, year: 2020 },
+    total: 34.00,
+    ship_to: 'JS Masher',
+    order_number: '114-3941689-8772232',
+    delivery_date: { delivered: 'Delivered', month: 'Aug', day: 8, year: 2020 },
+    delivery_message: '',
+    order: [{
+      product_name: 'Javascript for impatient programmers',
+      author: 'Rauschmayer, Dr. Axel',
+      return_window: 'Return window closed on Sep 7, 2020',
+      price: 31.55
+    }]
+  },
+  {
+    order_placed: { month: 'July', day: 19, year: 2020 },
+    total: 44.53,
+    ship_to: 'JS Masher',
+    order_number: '113-9984268-1280257',
+    delivery_date: { delivered: 'Delivered', month: 'Jul', day: 20, year: 2020 },
+    delivery_message: 'Your package was delivered. It was handed directly to a resident.',
+    order: [{
+      product_name: 'The Timeless Way of Building',
+      author: 'Alexander, Christopher',
+      return_window: 'Return window closed on Aug 19, 2020',
+      price: 41.33
+    }]
+  },
+  {
+    order_placed: { month: 'July', day: 4, year: 2020 },
+    total: 17.22,
+    ship_to: 'JS Masher',
+    order_number: '114-2875557-9059409',
+    delivery_date: { delivered: 'Delivered', month: 'Jul', day: 7, year: 2020 },
+    delivery_message: 'Your package was delivered. It was handed directly to a resident.',
+    order: [{
+      product_name: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter ' +
+        'for WII U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
+      author: 'Alexander, Christopher',
```

Comment:
I don't think that Christopher Alexander is the author of Gamecube controller adapters 🤔

Reply:
Oops that is embarrassing